File: AVL_Geometry/frmInfo.vb
Public Class frmInfo
    Private Sub frmInfo_FormClosing(sender As Object, e As FormClosingEventArgs) Handles MyBase.FormClosing
        If e.CloseReason <> CloseReason.None Then
            frmMain.RestartConsoleToolStripMenuItem.PerformClick()
        End If
    End Sub

    Private Sub frmInfo_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        lb1.Dock = DockStyle.Fill
        Me.Icon = frmMain.Icon
        Me.Left = Screen.PrimaryScreen.WorkingArea.Width - Me.Width
    End Sub
End Class

File: AVL_Geometry/frmInfo.cs
using System;
using System.Windows.Forms;

namespace AERO_Console
{
    public partial class frmInfo
    {
        public frmInfo()
        {
            InitializeComponent();
        }
        private void frmInfo_FormClosing(object sender, FormClosingEventArgs e)
        {
            if (e.CloseReason != CloseReason.None)
            {
                My.MyProject.Forms.frmMain.RestartConsoleToolStripMenuItem.PerformClick();
            }
        }

        private void frmInfo_Load(object sender, EventArgs e)
        {
            Icon = My.MyProject.Forms.frmMain.Icon;
            Left = Math.Max(20, Screen.PrimaryScreen.WorkingArea.Width - Width - 30);
            Top = Math.Max(40, Screen.PrimaryScreen.WorkingArea.Height - Height - 80);

            var theme = UI.Theme.Current;
            UI.Theme.UseImmersiveDarkMode(Handle, theme.IsDark);
            theme.ApplyTo(this);
            theme.StyleCard(pnlMain, UI.Theme.SpacingMd);
            theme.StyleButton(btnClose);
            lblTitle.ForeColor = theme.Accent;
            lblViewHeader.ForeColor = theme.Accent;
            lblZoomHeader.ForeColor = theme.Accent;
            theme.StyleBadge(lblNote);
        }
    }
}